mapsforge-map/src/main/java/org/mapsforge/map/layer/hills/AThreadedHillShading.java

```diff
@@ -553,7 +553,52 @@ protected int getStrideFactor(int inputAxisLen, int outputAxisLen) {
 
     @Override
     protected byte[] convert(InputStream inputStream, int dummyAxisLen, int dummyRowLen, int padding, int zoomLevel, double pxPerLat, double pxPerLon, HgtFileInfo hgtFileInfo) throws IOException {
-        return doTheWork(hgtFileInfo, false, padding, zoomLevel, pxPerLat, pxPerLon);
+        return doTheWork_(hgtFileInfo, false, padding, zoomLevel, pxPerLat, pxPerLon);
+    }
+
+    /**
+     * @param hgtFileInfo   HGT file info
+     * @param isHighQuality When {@code true}, a unit element is 4x4 data points in size instead of 2x2, for better interpolation possibilities.
+     * @param padding       Padding of the output, useful to minimize border interpolation artifacts (no need to be larger than 1)
+     * @param zoomLevel     Zoom level (to determine shading quality requirements)
+     * @param pxPerLat      Tile pixels per degree of latitude (to determine shading quality requirements)
+     * @param pxPerLon      Tile pixels per degree of longitude (to determine shading quality requirements)
+     * @return
+     */
+    protected byte[] doTheWork_(final HgtFileInfo hgtFileInfo, boolean isHighQuality, int padding, int zoomLevel, double pxPerLat, double pxPerLon) {
```

Comment:
@Sublimis `doTheWork` and `doTheWork_` are confusing.
Please avoid the `_` and name the methods differently.
`doTheWork` can become private as is not called elsewhere.

Reply:
No privacy in a non-final class, sorry